src/apps/AdminApp/AdminRoot.js

```diff
@@ -528,7 +533,8 @@ class AdminRoot extends Component {
         this.sendCommandMessage(command_type);
         return;
 
-        const {gateways, feed_user, current_janus, current_room, command_status, gdm} = this.state;
+        /* TEMPORARY NOT USED - DELETE IF NOT NEEDED */
```

Comment:
remove the "delete" part.
Premoderation is a feature yet to have been used. 
ok to comment out code but no need to promote it's ultimate removal as of now

Reply:
Done.

Reply:
Done.